k8s/namespaces/monitoring/kube-prometheus-stack/monitoring.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Since you've moved this file to monitoring, it would be better to call it namespace.yaml